.github/workflows/publish.yml

```diff
@@ -52,9 +49,9 @@ jobs:
           echo "##[set-output name=version;]$(echo ${GITHUB_REF#refs/tags/})"
 
       - name: Build apisix RPM Package
-        if: ${{ startsWith(steps.tag_type.outputs.version, 'apisix/') }}
+        if: ${{ startsWith(steps.tag_type.outputs.version, 'apisix/') || startsWith(steps.tag_type.outputs.version, 'build-all/') }}
```

Comment:
Why you add this condition?

Reply:
this must have been for debugging. I removed it from everywhere. @tao12345666333